app/src/main/java/dev/arkbuilders/arkmemo/ui/viewmodels/NotesViewModel.kt

```diff
@@ -164,6 +173,7 @@ class NotesViewModel
             note: Note,
             parentResId: ResourceId? = null,
         ) {
+            Log.d(TAG, "add note with title: ${note.title} resId: ${note.resource?.id}")
             val notes = this.notes.value.toMutableList()
             note.resource?.let {
                 notes.removeIf { it.resource?.id == parentResId ?: note.resource?.id }
```

Comment:
The right hand side should be extracted into a variable to improve readability

Reply:
No problem, btw Android Studio complains about clarifying parantheses on the same line too